state_base.py

```diff
@@ -28,19 +28,24 @@ class StateBase(Persistence, GroupBy, Block):
     def __init__(self):
         super().__init__()
         self._initial_state = None
+        self._changed_state = False
```

Comment:
It still looks to me like this block is going to fail to load from persistence during configure. Try running this block for real to witness that behavior.